### ISSUE TITLE
PHP 7.1: mcrypt_create_iv() is deprecated

### ISSUE THREAD

**Issue by HerrAugust** [philios33/PHP-AES-File-Encryption#4]
Error arises at line 27 of MCryptAES256Implementation.php

Online solutions suggest random_bytes()...

**Comment by ghost** [comment]
Yes. I think this project is dead though. The files has not been updated since 2015.
